settings/DevHome.Settings/Views/PreferencesPage.xaml

```diff
@@ -26,44 +27,23 @@
-                                        <xaml:ElementTheme>Light</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                                <RadioButton
-                                    x:Uid="Settings_Theme_Dark"
-                                    Command="{x:Bind ViewModel.SwitchThemeCommand}"
-                                    GroupName="AppTheme"
-                                    IsChecked="{x:Bind ViewModel.ElementTheme, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter=Dark, Mode=OneWay}">
-                                    <RadioButton.CommandParameter>
-                                        <xaml:ElementTheme>Dark</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                                <RadioButton
-                                    x:Uid="Settings_Theme_Default"
-                                    Command="{x:Bind ViewModel.SwitchThemeCommand}"
-                                    GroupName="AppTheme"
-                                    IsChecked="{x:Bind ViewModel.ElementTheme, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter=Default, Mode=OneWay}">
-                                    <RadioButton.CommandParameter>
-                                        <xaml:ElementTheme>Default</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                            </StackPanel>
-                        </labs:SettingsCard>
-                    </labs:SettingsExpander.Items>
-                </labs:SettingsExpander>
+                <labs:SettingsCard x:Uid="Settings_Theme">
+                    <labs:SettingsCard.HeaderIcon>
+                        <FontIcon Glyph="&#xE790;" />
+                    </labs:SettingsCard.HeaderIcon>
+                    <ComboBox x:Name="ThemeSelectionComboBox"
+                                      SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.ElementTheme, Converter={StaticResource EnumToIntConverter}}">
+                        <ComboBoxItem x:Uid="Settings_Theme_Default" />
+                        <ComboBoxItem x:Uid="Settings_Theme_Light" />
+                        <ComboBoxItem x:Uid="Settings_Theme_Dark" />
+                        <i:Interaction.Behaviors>
+                            <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.SwitchThemeCommand}" 
+                                                        CommandParameter="{Binding SelectedIndex, ElementName=ThemeSelectionComboBox}" />
+                            </ic:EventTriggerBehavior>
+                        </i:Interaction.Behaviors>
+                    </ComboBox>

```

Comment:
Current approach works, but could be simplified as suggested below, allowing us to eliminate the dependency on a converter (which will be hard to maintain if we add a supported types list in it), and relax the constraint for listing the combobox items based on the values order.

```suggestion
                        <ComboBoxItem x:Uid="Settings_Theme_Default" Tag="{x:Bind ui:ElementTheme.Default}" />
                        <ComboBoxItem x:Uid="Settings_Theme_Light" Tag="{x:Bind ui:ElementTheme.Light}" />
                        <ComboBoxItem x:Uid="Settings_Theme_Dark" Tag="{x:Bind ui:ElementTheme.Dark}" />
                        <i:Interaction.Behaviors>
                            <ic:EventTriggerBehavior EventName="SelectionChanged">
                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.SwitchThemeCommand}" 
                                                        CommandParameter="{x:Bind ((ComboBoxItem)ThemeSelectionComboBox.SelectedItem).Tag, Mode=OneWay}" />
                            </ic:EventTriggerBehavior>
                        </i:Interaction.Behaviors>
                    </ComboBox>
```

Reply:
I'm not sure how to get out of using the converter here -- the suggested code works for changing theme, but without the converter we can't start the combobox out with the correct item selected. I was hoping something like 
```
                    <ComboBox x:Name="ThemeSelectionComboBox"
                              SelectedValuePath="Tag"
                              SelectedValue="{x:Bind Mode=OneTime, Path=ViewModel.ElementTheme}">
```
would work, but it doesn't. Do you know what I'm doing wrong?

Reply:
Gotcha, if the initial value is not configurable from WinUI XAML, we may be able to configure this from code-behind or alternatively we can store the enum values in the view-model as a property, along with an observable property for the selected index (using two way binding).